common/religion/religions/naarism.txt

```diff
@@ -24,6 +24,20 @@
 	doctrine = doctrine_deviancy_crime
 	doctrine = doctrine_witchcraft_crime
 
+	#Magic
+	doctrine = doctrine_light_magic_approved
+	doctrine = doctrine_shadow_magic_crime
+	doctrine = doctrine_disorder_magic_crime
+	doctrine = doctrine_order_magic_accepted
+	doctrine = doctrine_life_magic_ignored
+	doctrine = doctrine_death_magic_crime
+	doctrine = doctrine_elemental_spirit_magic_accepted
+	doctrine = doctrine_elemental_decay_magic_shunned
+	doctrine = doctrine_elemental_fire_magic_accepted
+	doctrine = doctrine_elemental_water_magic_accepted
+	doctrine = doctrine_elemental_air_magic_accepted
+	doctrine = doctrine_elemental_earth_magic_accepted
```

Comment:
`lightbound` should have all magics except Light set to Crime.

Reply:
Done